### PR TITLE
fix: delete unused e2e test files

### PR DESCRIPTION
**Pull request by sysvm**
### Description

 Delete unused e2e test files

### Rationale

N/A

### Example

N/A

### Changes

N/A

### CODE REVIEW

**Review by joeylichang — APPROVED** [review]

File: .github/workflows/e2e-test.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ jobs:
       - name: Run Greenfield-SPs
         run: |
           bash ./deployment/localup/localup.sh --generate $GITHUB_WORKSPACE/greenfield/sp.json ${{ env.MYSQL_USER }} ${{ env.MYSQL_PASSWORD }} ${{ env.MYSQL_IP }}:${{ env.MYSQL_PORT }}
+          bash ./deployment/localup/localup.sh --gene2e $GITHUB_WORKSPACE/greenfield/sp.json
```
Comment (will-2012) [inline .github/workflows/e2e-test.yml:75]: This cmd is only used in the integration test, why put it here?

Comment (sysvm) [reply .github/workflows/e2e-test.yml:75]: This cmd belongs to SP. You said it should be moved to `Run Greenfield E2E Test` job?

Comment (will-2012) [reply .github/workflows/e2e-test.yml:75]: Yes, because Run-SP does not depend on it.

Comment (sysvm) [reply .github/workflows/e2e-test.yml:75]: ok

**Review by will-2012 — APPROVED** [review]

**Pull request merged**